meson.build

```diff
@@ -1057,6 +1057,7 @@ executable('systemd',
                            libselinux,
                            libmount,
                            libblkid],
+           install_rpath : '$ORIGIN',
```

Comment:
The systemd binary itself needs the explicit path here. To resolve `$ORIGIN` `/proc` is needed. And that's not mounted if systemd is used without initrd. I don't know if there are any other programs in `/usr/lib/systemd` that might be executed early.